.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "@patternfly/eslint-config-elements"
```

Comment:
Rules are marking 'extra' `{` as an error, would prefer if the project allowed folks that prefer their `{` or prefer to leave out the unnecessary ones.

Chat thread: https://chat.google.com/room/AAAAQFSmB88/CTIQyQNE3W4

Reply:
I set this rule like you asked :)

Please tell me when is a convenient time for you and i'll rebase/force push again